tests/encode/test1_suit/CMakeLists.txt

```diff
@@ -9,51 +9,25 @@ cmake_minimum_required(VERSION 3.13.1)
 set(PROJECT_NAME test1_suit)
 include(../../cmake/test_template.cmake)
 
-set(py_command_e
+set(py_command
   zcbor
   -c ${CMAKE_CURRENT_LIST_DIR}/../../cases/manifest3.cddl
   code
-  --output-cmake ${PROJECT_BINARY_DIR}/manifest3_encode.cmake
+  --output-cmake ${PROJECT_BINARY_DIR}/manifest3.cmake
   -t
   SUIT_Outer_Wrapper
   SUIT_Command_Sequence
   SUIT_Authentication_Wrapper
-  -e
-  --oc ${PROJECT_BINARY_DIR}/src/manifest3_encode.c
-  --oh ${PROJECT_BINARY_DIR}/include/manifest3_encode.h
```

Comment:
So this info is now grabbed from the --output-cmake name? IMO that feature is not clear from the help text.

Reply:
Good point, added explanation in a separate commit.